bin/www

```diff
@@ -0,0 +1,8 @@
+const app = require('../index.js');
```

Comment:
Remove bin/www directories. This codebase is not serving web pages, so no need for a directory structure such as `bin/www`. Stick to the basic project directories/files structure, and let's have the application entry file at the root directory.

Reply:
Having the server live separately from the express application has nothing to do with serving web pages. It's actually best practice to separate your 'Express' definition to at least two files: the API declaration (app.js) and the networking concerns (WWW).
This allows testing the API in-process, without performing network calls, with all the benefits that it brings to the table: fast testing execution and getting coverage metrics of the code. It also allows deploying the same API under flexible and different network conditions. Bonus: better separation of concerns and cleaner code

Reply:
You could have a `servee.js` or `app.js` and `index.js` at the root directory, which would accomplish the same thing than having depths of directories such as `bin/www`.

Reply:
what bad effect does having a bin/www file have on our codebase if there isn't why are we not following it since it's an accepted way of doing this